### ISSUE TITLE
Textual boundaries are missing from the Master Details pane

### ISSUE THREAD

**Issue by chlane** [vmware/clarity#3889]
Test URL: https://datagrid-detail-pane.surge.sh/datagrid/detail
Clarity Version: v2.0.0-rc.1
Angular Version: 8.0.0-rc.1
OS: any
Browser: any
Screen Reader: any

WCAG 2.1 Guideline: 
Success Criterion 1.3.2 Meaningful Sequence
(Level A)

Description:
The Master Details pane is implemented as a dialog with textual boundaries but those boundaries are currently empty as seen below.

<div class="clr-sr-only"></div>

Screen reader users will not hear anything when they read these elements.

Steps to Reproduce:
- Using a screen reader go to https://datagrid-detail-pane.surge.sh/datagrid/detail
- Open the Master Details pane
-  Read the entire dialog and observe that there are no textual boundaries at the beginning or end of the dialog

Recommendation:
Populate the elements used for the textual boundaries with descriptive text as seen below.

<div class="clr-sr-only">
Beginning of details pane, activate the close button to return to the data grid
</div>

**Comment by gnomeontherun** [comment]
This should be implemented in the final version. The URL provided is outdated. 

You can verify by going to https://deploy-preview-3797--vmware-clarity.netlify.com/dev/ and Datagrid->Detail page.

**Comment by chlane** [comment]
This should be implemented in the final version.  Follow the steps below to see the fix in the dev version

- Go to https://deploy-preview-3797--vmware-clarity.netlify.com/dev/
- Then go to Datagrid->Detail page.
-  Open the details pane and check the HTML for textual boundaries

**Comment by gnomeontherun** [comment]
Closing as resolved.
